onevizion/trackor.py

```diff
@@ -512,7 +527,7 @@ def UploadFile(self, trackorId, fieldName, fileName, newFileName=None):
 		self.request = self.OVCall.request
 
 		Message(URL,2)
-		Message("FileName: {FileName}".format(FileName=fileName),2)
+		Message("FileName: {FileName}".format(FileName=newFileName if byteString else fileName),2)
```

Comment:
I think we need to log both the filename and the file path.

In this implementation, either the name or the path is logged. This is weird

Reply:
Done